.github/workflows/test-application.yaml

```diff
@@ -131,6 +131,14 @@ jobs:
                     - 3306:3306
                 options: --health-cmd="mysqladmin ping" --health-interval=10s --health-timeout=5s --health-retries=5
 
+            mariadb:
+                image: mariadb:10.3
```

Comment:
@chirimoya should we add mariadb to our test matrix? Then I would go by adding here a 10.7 version.